builder/action.go

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2022 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Imported from Camel-k
+ */
+package builder
```

Comment:
Please use my repo instead. Let it be the source of truth. This way, we can make it a standalone library to be used in the near future.

Reply:
Ok, my understand was to use the poc as example, I'll polish the pr using the repo as a library